benchmarks/src/main/java/org/questdb/StringNullCheckBenchmark.java

```diff
@@ -94,12 +94,8 @@ public void globalSetup() throws Exception {
             try (SqlCompilerImpl compiler = new SqlCompilerImpl(engine)) {
                 compiler.compile("drop table if exists x", sqlExecutionContext);
                 compiler.compile("create table x as (select" +
-                        (a_nullFreq != 0
-                                ? " rnd_str(100, 70, 140, " + a_nullFreq + ") string_value,"
-                                : " rnd_str('', 'a', 'aa') string_value,") +
-                        (a_nullFreq != 0
-                                ? " rnd_varchar(70, 140, " + a_nullFreq + ") varchar_value,"
-                                : " rnd_varchar('', 'a', 'aa') varchar_value,") +
+                        " rnd_str(70, 140, " + a_nullFreq + ") string_value," +
```

Comment:
The goal of previous logic was to stress the case of the empty string, which is a special case for the JIT code. Instead of simplifying, we probably have to complexify the code to support even more configurations.

Reply:
It wasn't using empty strings only. We could change the low boundary for the rnd functions to 0 instead of 70. This way empty strings will be occasionally inserted.

Reply:
It was stressing the specific pattern where it's unpredictable whether you'll have to load the data header or not. That's why it had lots of empty strings, but mixed with non-empty ones. Ultimately this is quite similar to a mix with null values, but since that's not an obvious fact, I considered it relevant to have that benchmark type for reference.

Reply:
From the benchmark perspective, an empty string is the same as null. Having an empty string and a few tiny strings doesn't represent what end users do. In such cases, they should use a symbol, not a string. We could add more cases like `rnd_str(0, 140, " + a_nullFreq + ")`, but I doubt they'll change the overall picture. WDYT?

Reply:
Yes, the difference between empty string and null won't show up on the benchmarks. For me it was more about having a proof of that captured in a benchmark -- i wrote it when I was still learning about QuestDB.